scripts/background-scripts/urlbar.js

```diff
@@ -0,0 +1,81 @@
+'use strict'
+
+var placeholders = [];
+var url = 'https://raw.github.com/bwinton/whimsy/gh-pages/urlbar-sayings.txt';
+
+setDefaults([
+  'Where do you want to go today?',
+  'Just type ”google.com“.  You know you’re going to.',
+  'Let’s do this thing!',
+  'Hey, I wonder what we should have for lunch?',
+  'Where to, boss?',
+  'I hear Facebook is nice this time of year…',
+  'You know you can search from here, right?',
+  'Have you thought about trying Private Browsing Mode?',
+  'You are in a maze of twisty web pages, all alike.',
+  'Hi!  My name is Url.'
+]);
+
+function setDefaults(defaults){
+  placeholders = defaults;
+}
+function loadPlaceholders() {
+  //load sayings from url
+  return new Promise((resolve, reject) => {
```

Comment:
We should probably use the [Fetch API](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch) instead of this.  It's newer, _and_ easier to use…  :)